sklearnex/tests/test_config.py

```diff
@@ -37,3 +44,54 @@ def test_set_config_works():
     assert config["allow_fallback_to_host"]
     assert config["assume_finite"]
     sklearnex.set_config(**default_config)
+
+
+gpu_mark = pytest.mark.skipif(
+    not _is_dpc_backend,
+    reason="gpu_blas_compute_mode is only supported with the dpc backend",
+)
+
+
+@pytest.mark.parametrize(
+    "setting",
+    [
+        pytest.param("standard", marks=gpu_mark),
+        pytest.param("FORCE_ALTERNATE", marks=gpu_mark),
+        pytest.param(
+            ["FLOAT_TO_BF16", "float_to_bf16x2", "float_to_bf16x3"], marks=gpu_mark
+        ),
+        pytest.param(
+            "float_to_bf16,float_to_bf16x2,float_to_bf16x3",
+            marks=gpu_mark,
+        ),
+        pytest.param("any", marks=gpu_mark),
+    ],
+)
```

Comment:
Make sure that you have correct logs/names for tests.
[setting2] is not informative [link](https://dev.azure.com/daal/daal4py/_build/results?buildId=30631&view=logs&j=517fe804-fa30-5dc2-1413-330699242c05&t=6200c9a9-fd17-566a-4b28-c9cb31ff00c5&l=1359)

```
s/sklearnex/tests/test_config.py::test_set_gpu_blas_compute_mode[standard] PASSED [ 18%]
s/sklearnex/tests/test_config.py::test_set_gpu_blas_compute_mode[FORCE_ALTERNATE] PASSED [ 19%]
s/sklearnex/tests/test_config.py::test_set_gpu_blas_compute_mode[setting2] PASSED [ 19%]
s/sklearnex/tests/test_config.py::test_set_gpu_blas_compute_mode[float_to_bf16,float_to_bf16x2,float_to_bf16x3] PASSED  [ 19%]
s/sklearnex/tests/test_config.py::test_set_gpu_blas_compute_mode[any] PASSED [ 20%]
```